examples/feed-ingress-deployment-merlin.yml

```diff
@@ -0,0 +1,184 @@
+
+        resources:
+          requests:
+            cpu: "1"
+            memory: 300Mi
+          limits:
+            memory: 300Mi
+
+        ports:
+        - hostPort: 80
+          containerPort: 80
+          name: ingress
+          protocol: TCP
+        - hostPort: 8081
+          containerPort: 8081
+          name: ingress-health
+          protocol: TCP
+        # Health port of the controller.
+        - containerPort: 12082
+          name: health
+          protocol: TCP
+
+        args:
+        # Ingress nginx port that frontend will direct traffic towards.
+        - -ingress-port=80
+
+        # Health port on nginx, used by frontend to determine health.
+        - -ingress-health-port=8081
+
+        # Default security whitelist for ingress. Can be overridden with the sky.uk/allow annotation.
+        - -ingress-allow=10.0.0.0/8
+
+        # Health port for the controller to respond on.
+        - -health-port=12082
+
+        # Log level of nginx. Recommended to leave at error, or set to crit if too much spam.
+        - -nginx-loglevel=error
+
+        # How often to reload nginx if needed. Setting too low can cause 504s from the frontend in the case of heavy
+        # ingress updates.
+        - -nginx-update-period=5m
+
+        # Use pushgateway for prometheus metrics. Optional - metrics available at /metrics on health port.
+        - -pushgateway=mypushgateway.com
+        - -pushgateway-interval=20
+        - -pushgateway-label=k8s_cluster=dev
+        - -pushgateway-label=environment=dev
+        - -pushgateway-label=version=v1.0.2
+
+        # Specify merlin as the frontend component
+        - -registration-frontend-type=merlin
+
+        # gRPC endpoint
+        - -merlin-endpoint=dns:///merlin-servers
+
+        # Virtual Service ID to attach to
+        - -merlin-service-id=my-virtual-service
+
+        # Real server IP to associate with virtual service - the IP of the ingress node.
+        - -merlin-instance-ip=$(INSTANCEIP)
+
+        # Real server port to associate with virtual service - the port of nginx.
+        - -merlin-instance-port=80
+
+        # Forward method that IPVS should use - pick route, masq, or tunnel.
+        - -merlin-forward-method=route
+
+        # Drain delay to bleed off connections when detaching.
+        - -merlin-drain-delay=60s
+
+        # Virtual IP to bind to the local interface for tunneling and direct routing. Usually the virtual service IP.
+        - -merlin-vip=10.10.10.1
+
+        # Interface to bind virtual IP to.
+        - -merlin-vip-interface=lo
+
+        # Each worker uses a full cpu, so scale up vertically on a box by increasing this value.
+        - -nginx-workers=1
+
+        # Connections*workers needs to be less than available ephemeral ports. Linux default is 60999-32768=28231.
+        - -nginx-worker-connections=20000
+
+        # Needs to be greater than the frontend idle timeout, see `aws_provisioning/kubernetes` for frontend value.
```

Comment:
It's just occurred to me that referencing `aws_provisioning/kubernetes` makes no sense in the context of Feed as a standalone application. We should change this as well.

Reply:
If it's referenced in other example files, we should change those too.

Reply:
We really should autogenerate these, or something.

Reply:
Anyways, I fixed it.